OpenRA.Mods.TS/UtilityCommands/LegacyTilesetImporter.cs

```diff
@@ -95,7 +100,7 @@ public void Run(ModData modData, string[] args)
 								var terrainType = s.ReadUInt8();
 								var rampType = s.ReadUInt8();
 
-								if (!terrainTypes.ContainsKey(terrainType))
+								if (terrainType < 0 || terrainType > 15)
```

Comment:
Maybe compare to ```terrainTypes.Length``` to avoid the magic constant. Also you appear to be testing if a byte can be negative.